techniques/html/H58.html

```diff
@@ -2,68 +2,70 @@
+      <h1>自然言語の変更を指定するために、言語属性を使用する</h1>
       <section id="important-information">
-         <h2>Important Information about Techniques</h2>
-         <p>See <a href="https://waic.jp/docs/WCAG21/Understanding/understanding-techniques">Understanding Techniques for WCAG Success Criteria</a> for important information about the usage of these informative techniques and how
-            they relate to the normative WCAG 2.1 success criteria. The Applicability section
-            explains the scope of the technique, and the presence of techniques for a specific
-            technology does not imply that the technology can be used in all situations to create
-            content that meets WCAG 2.1.
+         <h2>達成方法に関する重要な情報</h2>
+         <p>これらの達成方法 (参考) の使用法及び、それらが WCAG 2.1 達成基準 (規定) とどのように関係するかに関する重要な情報については、<a href="https://waic.jp/docs/WCAG21/Understanding/understanding-techniques">WCAG 達成基準の達成方法を理解する</a>を参照のこと。適用 (対象) セクションは、その達成方法の範囲について説明しており、特定の技術に関する達成方法があるからといって、WCAG 2.1 を満たすコンテンツを作成する際に、常にその技術が使用可能であるわけではない。
          </p>
       </section>
       <main>
          <section id="applicability">
-            <h2>Applicability</h2>
+            <h2>適用 (対象)</h2>
             <p>HTML</p>
-            <p>This technique relates to <span><a href="https://waic.jp/docs/WCAG21/Understanding/language-of-parts">Success Criterion 3.1.2: Language of Parts</a> (Sufficient)</span>.
+            <p>これは<span><a href="https://waic.jp/docs/WCAG21/Understanding/language-of-parts">達成基準 3.1.2: 一部分の言語</a> (十分な達成方法)</span> に関する達成方法である。
             </p>
          </section>
          <section id="description">
-            <h2>Description</h2>
-            <p>The objective of this technique is to clearly identify any changes in language on
-               a page by using the <code class="att">lang</code> attribute.
-            </p>
-            <p>Allowed values for the lang attribute are indicated in the resources referenced below.
-               Language tags use a primary code to indicate the language, and optional sub-codes
-               (separated by hyphen characters) to indicate variants of the language. For instance,
-               English is indicated with the primary code "en"; British English and American English
-               can be distinguished by using "en-GB" and "en-US", respectively. Use of the primary
-               code is important for this technique. Use of subcodes is optional but may be helpful
-               in certain circumstances.
+            <h2>解説</h2>
+            <p>この達成方法の目的は、<code class="att">lang</code> 属性を用いることで、ページ上で用いられている自然言語の変化を明示することである。
             </p>
+            <p>lang 属性に指定できる値については、以下に示す参考リソースに示されている。言語タグは、言語を表すプライマリーコードと、その言語が使用される地域や記述に用いる文字コードなどを表すサブコードから成っている。サブコードの指定は任意で、指定する場合にはプライマリーコードに続けてハイフンを記述し、その後に記述する。たとえば、プライマリーコード「en」は英語を示し、「en-GB」と「en-US」はそれぞれイギリス英語とアメリカ英語を示す。この達成方法において、プライマリーコードの使用は重要である。サブコードの使用は任意だが、状況によっては有用なものになり得る。            </p>
```

Comment:
（あまり嬉しくないですが、）一応原文に沿った形で訳出しました。
```suggestion
            <p>lang 属性に指定できる値については、以下に示す参考リソースに示されている。言語タグは、言語を表すプライマリーコードと、その言語の変形を表すオプションの (ハイフン文字で区切られる) サブコードを使用する。たとえば、英語はプライマリーコード「en」はで示される。イギリス英語とアメリカ英語はそれぞれ「en-GB」と「en-US」を用いて区別できる。この達成方法において、プライマリーコードの使用は重要である。サブコードの使用は任意だが、状況によっては有用なものになり得る。            </p>
```